ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__automated-release-stable-4.15-upgrade-from-stable-4.15.yaml

```diff
@@ -65,7 +65,7 @@ tests:
 - as: aws-ipi-fips-arm-mixarch-f1
   cron: 0 10 * * *
```

Comment:
@liangxia, we need to complete these in working hour, right?

Reply:
I'm thinking if we can manually update cron job settings, as we currently have 2 slices, if more jobs run in the daytime, we will hit quota issues.

Reply:
For automated release, we mainly use api to trigger this job.  And currently, we just let this job run everyday, to evaluate the pass ratio of the job and the test cases in it.